### PR TITLE
Use dynamic port number for integration test

### PR DESCRIPTION
**Pull request by reftel**
Also makes the tests get skipped insted of failing when run from outside of maven.

### CODE REVIEW

File: modules/swagger-jaxrs/pom.xml [hunk 1]
```diff
@@ -49,6 +49,26 @@
                 </configuration>
             </plugin>
             <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>build-helper-maven-plugin</artifactId>
+                <version>1.6</version>
```
Comment (frantuma) [inline modules/swagger-jaxrs/pom.xml]: it possibly makes sense using latest plugin version `3.0.0`

Comment (reftel) [reply modules/swagger-jaxrs/pom.xml]: Fixed. Thanks!

**Comment by webron** [comment]
Should this be migrated to the 2.0 branch too?

**Comment by frantuma** [comment]
applied also to 2.0 in #2535

**Pull request merged**